### ISSUE TITLE
After update 1.5.6-rc.11 and rc.13 KaKu- and Action-devices are not working anymore

### ISSUE THREAD

**Issue by hanek004** [athombv/homey#2219]
**Metadata**

* **Homey Version**: 1.5.6-rc.11
* **Homey Log Code**: ED192E9B4A
* **Smartphone Platform**: iPhone
* **Smartphone App Version**: 1.3.3 - Build 70

**If related to a Homey App:**
* **Homey App Version**: Action v2.3.0; KiKa v3.1.5
* **Homey App Log Code**: ???
* **Affected Devices (preferably product codes)**: Action (Eurodomest 972080), KaKu (AGDR-300, ACD-1000)

**What did you try to achieve?**

After update to rc 11 I noticed all flows switching Action- and KaKu-devices were not working. Manually switching the devices (i.e. via the Homey App, or Homey Desktop) is not working either. After a ptp (shorter ones of <3 minutes and longer one of >10 minutes) the devices work switching on/off only a few times (<5) and/or a short periode of time (<3 minutes). The same happens after restarting the 433/868 MHz microcontroller, and also the same happens after rebooting Homey.

**What did you expect as result?** 

I expected the devices to switch on/off, as they did before the Homey-update.

**What was the result?**

Devices don't switch and after restart controller / reboot Homey / ptp (>10 minutes) the devices work a couple of times / for a short while. After that the problem persists.

**Any other remarks**

I updated from the most recent previous release candidate.
Just updated to rc13 from rc11 and tested: also in this release candidate version the problem still exists.

**Comment by hanek004** [comment]
Sorry for the lack of more details, pressend ‘submit’ too early.
After resetting the 433Mhz-controller the KaKu- and Action-devices will work for a limited amount of time/times (very shortly / very few times). Problem occured directly after last update.

**Comment by Dijker** [comment]
Just edit above, no problem 
Firmware version and send a log after the problem occurs

**Comment by chrisboer** [comment]
@hanek004 I don't have issues with KaKu and also just updated towards 1.5.6 rc11

**Comment by nilsaw11** [comment]
Same here....  After a reboot I can send two commands the kiku devices and then it stop working...  Reboot and two commands to the kiku and it's  stop working .... So on..

**Comment by chrisboer** [comment]
@nilsaw11 also done about 4 times on an off no issues what type of KaKu do you have ?

**Comment by bvdbos** [comment]
Working for me. As @Dijker asked please fill the requested fields (especially, send the log-code through settings-system and include the log-code here). For troubleshooting 433 devices it's always advisable to pull the plug for a couple of minutes (though I'm not sure this will work in this case).

**Comment by bm480** [comment]
Also no problem here, after updating to to 1.5.6 rc11.

**Comment by nilsaw11** [comment]
@chrisboer : ,receivers: APA3-1500R, ACM-2300H  transmitter :ACDB-7000B. This is what i know, otherstuff is just buid-in behind swith.

**Comment by OskarBrink** [comment]
I have exactly the same problem since this weekend.
APA3-1500R, ACD1000, also my AWMR-230/210 and ACM-3000H2

**Comment by bvdbos** [comment]
@OskarBrink and @nilsaw11 : Did you do a Pull-The-Plug? Do you send mulitple 433 signals at once?

**Comment by hanek004** [comment]
@bvdbos : Tried both, so: (1) switching multiple devices at once in a flow, (2) switching devices manually (via iPhone-app and via Homey Desktop). In both cases the problem occurs as described, after restarting the controller / long ptp / reboot.

**Comment by nilsaw11** [comment]
@bvdbos Yes i pull the plug, restart de kaku app end switch off the app and than switch on the app.
only after a reboot from homey it will send some commands, the max is count to two.
note, just update to RC13, still no normal kaku working here :-(

**Comment by OskarBrink** [comment]
I have pulled the plug, first time it worked perhaps 2-3 times, after which it failed again.
I do not send multiple commands. Even trying to learn a device from scratch is not working. Neither from the remote control from KAKU nor by the device.
I am still on rc.11 - I will see to get the update to rc.13 later this evening.

**Comment by jeroenvollenbrock** [comment]
Closing as duplicate of #2218
